cime_config/acme/machines/taskmaker.pl

```diff
@@ -81,7 +81,7 @@
 # add the list of ALL *xml fields to a hash
 #-------------------------------------------------------------------------------
 my $parser = XML::LibXML->new( no_blanks => 1);
-my $caseroot = $ENV{CASEROOT};
```

Comment:
Don't like passing things by env. My change is a little hacky, but should work fine until taskmaker is ported to python.